cmd/ctr/commands/images/images.go

```diff
@@ -212,17 +212,20 @@ var checkCommand = cli.Command{
 			return err
 		}
 		defer cancel()
-		var (
-			contentStore = client.ContentStore()
-			tw           = tabwriter.NewWriter(os.Stdout, 1, 8, 1, ' ', 0)
-		)
-		fmt.Fprintln(tw, "REF\tTYPE\tDIGEST\tSTATUS\tSIZE\tUNPACKED\t")
+
+		var contentStore = client.ContentStore()
 
 		args := []string(context.Args())
 		imageList, err := client.ListImages(ctx, args...)
 		if err != nil {
 			return errors.Wrap(err, "failed listing images")
 		}
+		if len(imageList) == 0 {
+			return errdefs.ErrNotFound
```

Comment:
This isn't a good return error when there are legitimately no images, perhaps we need better support for a `-q` option to make these more easily scriptable

Reply:
fair.. updated the usage, and added a debug log out for when there is nothing to check ..

Reply:
and added the -q option...